packages/amazonq/.changes/next-release/Bug Fix-037e19bd-95f5-4cdf-8408-6eb273ff7ebd.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Stuck on `Uploading code...` for /dev and /doc in some multi-root workspaces"
```

Comment:
One more small thing: should we clarify the case when this happens? Something like:
```
/dev and /doc: Duplicate file detection causes infinite 'Uploading code...' loop
```